public/app/features/search/page/SearchPage.tsx

```diff
@@ -88,6 +88,14 @@ export default function SearchPage() {
     setSearchSelection(updateSearchSelection(searchSelection, !current, kind, [uid]));
   };
 
+  const onUpdateList = async () => {
+    // clean up search selection
+    setSearchSelection(clearSearchSelection(searchSelection));
+    // trigger again the search to the backend
+    // TODO: the backend is not returning the updated value :(
+    onQueryChange(inputValue);
```

Comment:
I think the backend might need to re-index or something (I say this but I don't really know how bluge works or anything). Restarting grafana-server fixes it for me

Reply:
Yes, I asked and the issue is known and being worked on :)